src/Components/Hero/Hero.scss

```diff
@@ -0,0 +1,28 @@
+@import 'src/scss/variables';
+
+.hero {
+  position: relative;
+  height: 396px;
+  padding: 0 #{$layout-side-padding};
+  background-image: url('../../img/header.png');
```

Comment:
I would use absolute path

Reply:
Not sure, how I can use it here, to be honest.
Relative one is possible as well.
https://developer.mozilla.org/en-US/docs/Web/CSS/url